test/e2e/data/rancher/rancher-setting-patch.yaml

```diff
@@ -15,3 +15,9 @@ kind: Setting
 metadata:
   name: eula-agreed
 value: "2023-08-16T20:39:11.062Z"
+---
+apiVersion: management.cattle.io/v3
+kind: Setting
+metadata:
+  name: agent-tls-mode
+value: "system-store"
```

Comment:
By reading the linked issue and PR again, it seems we need to set this to "strict" if using Rancher 2.9

Reply:
`system-store` will surface the error on the cluster, which is better for debugging purposes.